packages/bpk-component-calendar/src/BpkCalendarDate-test.js

```diff
@@ -40,6 +40,15 @@ describe('BpkCalendarDate', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('should render a disabled and blocked button', () => {
+    const tree = renderer
+      .create(
+        <BpkCalendarDate date={new Date(2010, 1, 15)} disabled isBlocked />,
```

Comment:
What's the difference between `disabled` and `isBlocked`? Can we use one prop to do both?

Reply:
It's a little complex due to the fact that we allow components to be used both in isolation and as wrapped components that are provided specific props.

BpkCalendarDate takes both a `buttonProps` implicit `rest` that can be spread onto the button, and a separate specific `isBlocked` prop.

`disabled` is used in some tests, but isn't explicitly part of the API. Rather, `isBlocked` is used to set `disabled` - `disabled={isBlocked}`. The test shows how `disabled` can also be passed in as a singular prop.

However, `Week.js` provides `isBlocked` to its `DateComponent`, with `DateComponent` being passed in as a prop. In all cases I've seen this `DateComponent` is `BpkCalendarDate`, but it could technically be anything. From the existing code it looks like `isBlocked` is intended to be the declarative prop passed around.

So, to answer the question - we _almost_ do use just one - if only `isBlocked` is provided it also sets `disabled`. The API allows for unsetting this using `isBlocked disabled={false}` if someone really wanted to. The test is a little redundant having both set here - `isBlocked` will set `disabled=true` and then `disabled` will set `disabled=true`. It does represent something possible with the API, but unlikely. 

I could make the test only pass `isBlocked`, or could pass both to highlight it is possible - do you have a preference? Possibly the former.

Reply:
I'd probably prefer just `isBlocked` as the `disabled` prop is redundant